encoding/resp/resp.go

```diff
@@ -11,6 +11,51 @@ var (
 	ErrInvalidProtocol = errors.New("invalid protocol")
 )
 
+type Type byte
+
+const (
+	SimpleString Type = '+'
+	Error        Type = '-'
+	Integer      Type = ':'
+	BulkString   Type = '$'
+	Array        Type = '*'
+)
+
+// The 'Value' design is referenced from https://github.com/tidwall/resp
+type Value struct {
```

Comment:
Is the design of `Value` really necessary, it is a conflict style with the original design. Maybe we should think again and supply more reasons to change the design.

Reply:
I will change the PR to use the original way to construct response.